source/manual/2nd-line.html.md

```diff
@@ -70,6 +68,14 @@ If your working patterns are not compatible with a 9.30am-5.30pm shift, let the
 
 If you do not work a 5-day week, please talk to your delivery manager to arrange cover with another developer on your team.
 
+### Away days and all-staff events
+
+Before attending an all-staff event, team away-day or any other event that could keep you away from your laptop for long periods at a time: try to swap your shift with someone who is _not_ attending ([see above](#shift-swaps-working-patterns-and-sickness)).
```

Comment:
Could we add that, it's not possible to swap we recommend that people keep their laptops on them and that tether from their (work) phone for internet access if necessary?

Reply:
Ah, it wasn't clear that they'd be expected to work on the incident whilst at an away day. So perhaps expand "and are prepared to drop what you're doing if anything urgent comes in" to append something like " to work on the incident.

So yes, you can go to the away day, but you must be able to sign in and work even if it's from a park bench.

Reply:
Added a tweak 👍